commit 9b37f756f54f41247d398879bffb6983de53af66
Author: intrigeri <intrigeri@boum.org>
Date:   Sat May 3 14:08:04 2014 +0000

    Exclude tmpfs filesystems from df output in the sys handler (Closes: Debian#745818).

diff --git a/handlers/sys.in b/handlers/sys.in
index ae86f17..1d47977 100644
--- a/handlers/sys.in
+++ b/handlers/sys.in
@@ -303,7 +303,7 @@ STATUS="Checking available memory:"
 catifexec "/usr/bin/free"
 
 STATUS="Checking free disk space:"
-catifexec "/bin/df" "-al"
+catifexec "/bin/df" "-al --exclude-type=tmpfs"
 
 STATUS="Collecting what services run at what run level:"
 if [ $os = "redhat" ]; then
